src/components/Common/StatusBadgeMini/index.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  BellIcon,
+  CheckIcon,
+  ClockIcon,
+  MinusSmIcon,
+} from '@heroicons/react/solid';
+import { MediaStatus } from '@server/constants/media';
+
+interface StatusBadgeMiniProps {
+  status: MediaStatus;
+  is4k?: boolean;
+  indicatorIcon?: React.ReactNode;
```

Comment:
Don't forget to remove this from the type.

Reply:
woops